### PR TITLE
Add a watermark to draft PDFs

### PR DESCRIPTION
**Pull request by vrajmohan**
The user can download a PDF version of their submission at any time. To
avoid confusion, these PDFs will carry a DRAFT watermark. The PDF of the
final submission will not have this watermark.

This fixes eregs/notice-and-comment#124.

### CODE REVIEW

File: regulations/tasks.py [hunk 1]
```diff
@@ -76,21 +77,35 @@ def publish_tracking_number(response, key):
...
     return loader.render_to_string(
-        'regulations/comment.html', {'sections': sections})
+        'regulations/comment.html',
+        {'sections': sections, 'mark_as_draft': mark_as_draft}
+    )
 
 
 @contextlib.contextmanager
 def html_to_pdf(html):
+    print(html)
```
Comment (cmc333333) [inline regulations/tasks.py]: Stray `print`

**Comment by cmc333333** [comment]
Very cool. To confirm, if the doc is multiple pages long, the watermark will appear on each page?

**Comment by vrajmohan** [comment]
So I thought. On testing a multiple page PDF, it doesn't seem to work. Investigating...

File: regulations/templates/regulations/comment.html [hunk 2]
```diff
@@ -2,9 +2,14 @@
 
 <head>
   <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+
 </head>
 
 <body>
+  {% if mark_as_draft %}
+    <div class="watermark">
+  {% endif %}
```
Comment (cmc333333) [inline regulations/templates/regulations/comment.html]: Super duper minor, but if you need to make a change to fix conflicts, would this work?
```html
<body {% if mark_as_draft %}class="watermark"{% endif %}>
```

**Comment by cmc333333** [comment]
Excited to see it

**Pull request merged**